src/docs.md

```diff
@@ -47,6 +47,26 @@ Some of the new content includes:
 If you have questions or comments about the revamped site, [file an
 issue]({{site.repo.this}}/issues).
 
+{% comment %}
+# TODO (Not for publication)
```

Comment:
How about putting this in an issue (or our shared doc) instead of embedding this in the docs page?

Reply:
Yes, definitely. I was just throwing out ideas when preparing for our last stakeholder meeting.

Reply:
The to-do list is still in the doc.